nilearn/tests/test_signal.py

```diff
@@ -617,14 +617,14 @@ def test_clean_psc():
 
 
 def test_clean_zscore():
-    rng = np.random.RandomState(0)
+    rng = np.random.RandomState(42)
```

Comment:
When I was 42, I kept wondering what's special with this number --- and did not find.
This is no longer an issue now ;-)